ui/src/shared/components/time_series/TimeSeries.tsx

```diff
@@ -119,13 +130,12 @@ class TimeSeries extends Component<Props, State> {
       'cellType',
       'manualRefresh',
     ]
-
-    return (
+    const should =
```

Comment:
funky! i think i'd rather name the condition and then return that named condition that helps me understand what's being asserted, rather than using the namespace of the containing function (`shouldComponentUpdate`)